templates/partials/_navbar.html.twig

```diff
@@ -0,0 +1,72 @@
+{% block navbar %}
```

Comment:
you don't have to precise this block since you're including it inside the same block in your base file